File: frontend/dashboard/types/aspect-ratio.ts
interface AspectRatio {
  id: number;
  ratio: string;
  width: number;
  height: number;
}

interface CreateAspectRatioRequest {
  ratio: string;
  width: number;
  height: number;
}

interface UpdateAspectRatioRequest {
  ratio?: string;
  width?: number;
  height?: number;
}

File: dto/aspectratio.go
package dto

type AspectRatioResponse struct {
	ID     int    `json:"id"`
	Ratio  string `json:"ratio"`
	Width  int    `json:"width"`
	Height int    `json:"height"`
}

type CreateAspectRatioRequest struct {
	Ratio  string `json:"ratio"`
	Width  int    `json:"width"`
	Height int    `json:"height"`
}

type UpdateAspectRatioRequest struct {
	Ratio  *string `json:"ratio,omitempty"`
	Width  *int    `json:"width,omitempty"`
	Height *int    `json:"height,omitempty"`
}
